docs: Add issue and pull request templates

selbekk opened the pull request:
This pull request adds an issue template that asks some basic questions that will help the user write a descriptive issue.

I'm open for discussing the contents - but I think this is one of those things we improve by using it a lot.

Fixes #9

selbekk committed:
docs: Add issue template

This commit adds an issue template that asks some basic questions
that will help the user write a descriptive issue.

selbekk commented:
Not sure if we need a pull request template as well?

wkillerud reviewed (dismissed):
A good start :) It may seem odd if filing a question, but we'll work it out as we go.

selbekk commented:
Not sure how we would deal with usage questions. I can add a comment about wiping the template if you have a question.

selbekk committed:
docs: Add pull request template

This commit adds a pull request template, reminding contributors to
adhere to our code of conduct and contribution guidelines.

selbekk commented:
I added a pull request template as well. It's not really a template as much as it's a reminder to adhere to CONTRIBUTING.md guidelines.

antidecaf commented on PULL_REQUEST_TEMPLATE.md:
```diff
@@ -0,0 +1,14 @@
+Before submitting a pull request, please have a look through the CONTRIBUTING.md
+document. TL;DR:
+
+- Write proper commit messages tagged with the correct package and scope
```
Mention conventional commits specifically here, perhaps?

selbekk replied:
Sure. Do you have a suggestion as to how it should be worded?

antidecaf replied:
> Write proper commit messages tagged with the correct package and scope. We follow the [Conventional Commits](https://conventionalcommits.org/) convention.

... or some such. :)

selbekk commented:
@antidecaf ended up with something similar. Unfortunately, this is unparsed markdown, so you can't really have links. Hopefully this will be good enough - at least for a start :)
